static/script/devices/broadcastsource/stubbedsource.js

```diff
@@ -0,0 +1,90 @@
+/**
+ * @fileOverview Requirejs module containing the antie.widgets.hbbtvsource class.
+ *
+ * @preserve Copyright (c) 2013-2014 British Broadcasting Corporation
+ * (http://www.bbc.co.uk) and TAL Contributors (1)
+ *
+ * (1) TAL Contributors are listed in the AUTHORS file and at
+ *     https://github.com/fmtvp/TAL/AUTHORS - please extend this file,
+ *     not this notice.
+ *
+ * @license Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * All rights reserved
+ * Please contact us for an alternative licence
+ */
```

Comment:
Thanks :)

From: Tom Sadler <notifications@github.com<mailto:notifications@github.com>>
Reply-To: fmtvp/tal <reply@reply.github.com<mailto:reply@reply.github.com>>
Date: Tuesday, 8 December 2015 11:02
To: fmtvp/tal <tal@noreply.github.com<mailto:tal@noreply.github.com>>
Cc: Joel Howard <Joel.Howard@bbc.co.uk<mailto:Joel.Howard@bbc.co.uk>>
Subject: Re: [tal] IPTV-708 Implemented broadcast stub and fixed old base tv source tests (#319)


In static/script/devices/broadcastsource/stubbedsource.js<https://github.com/fmtvp/tal/pull/319#discussion_r46939983>:

> + *
> + * @license Licensed under the Apache License, Version 2.0 (the "License");
> + * you may not use this file except in compliance with the License.
> + * You may obtain a copy of the License at
> + *
> + *    http://www.apache.org/licenses/LICENSE-2.0
> + *
> + * Unless required by applicable law or agreed to in writing, software
> + * distributed under the License is distributed on an "AS IS" BASIS,
> + * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
> + * See the License for the specific language governing permissions and
> + * limitations under the License.
> + *
> + * All rights reserved
> + * Please contact us for an alternative licence
> + */


Good point Joel - I was meant to tell people about this...will spam everyone now.

—
Reply to this email directly or view it on GitHub<https://github.com/fmtvp/tal/pull/319/files#r46939983>.